apps/modernization-ui/src/apps/search/layout/result/list/SearchResultList.tsx

```diff
@@ -8,11 +8,18 @@ type Props<T> = {
 };
 
 const SearchResultList = <T,>({ results, render }: Props<T>) => {
-    return results.map((result, index) => (
-        <div className={styles.result} key={index}>
-            {render(result)}
+    return results && results.length > 0 ? (
+        results.map((result, index) => (
+            <div className={styles.result} key={index}>
+                {render(result)}
+            </div>
+        ))
+    ) : (
+        <div className="text-center">
+            <p>No results found.</p>
+            <p>Try refining your search.</p>
         </div>
```

Comment:
issue:  This is for story [CNFT1-2469](https://cdc-nbs.atlassian.net/browse/CNFT1-2469) that Alaap is working on.  The `SearchResults` component is meant to be displayed only if there are results to show.  The [SearchLayout](https://github.com/CDCgov/NEDSS-Modernization/blob/main/apps/modernization-ui/src/apps/search/layout/SearchLayout.tsx) is going to handle which component is displayed based on the out come of the search